packages/message-manager/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.0.1]
+### Changed
```

Comment:
It might make more sense to categorize this as a bug fix. If this is a feature or change in behavior, we'd probably want this to be a minor release. But bug fix makes sense in this case I think

```suggestion
### Fixed
```